src/CommonRLInterface.jl

```diff
@@ -47,7 +64,7 @@ end
 Base.convert(::Type{AbstractEnv}, env::CRL.AbstractEnv) = convert(RLBaseEnv, env)
 Base.convert(::Type{RLBaseEnv}, env::CRL.AbstractEnv) = RLBaseEnv(env, 0.f0)  # can not determine reward ahead. Assume `Float32`.
 
-get_state(env::RLBaseEnv) = CRL.observe(env.env)
+get_state(env::RLBaseEnv) = InformationStyle(env) === PERFECT_INFORMATION ? CRL.state(env.env) : CRL.observe(env.env)
```

Comment:
Hmm... I dont think this is the best thing to do. I think it should always be `observe`. All of your algorithms use `get_state` to get information to make a decision on, right?

If someone implemented a partially observable environment in `CommonRLInterface`, they would expect decision to be made only on the observation.
```suggestion
get_state(env::RLBaseEnv) = CRL.observe(env.env)
```

Reply:
> If someone implemented a partially observable environment in CommonRLInterface, they would expect decision to be made only on the observation.

You are correct here. Originally I thought only `CRL.state` is provided for POMDP environments. After checking the doc again, I find that 'CRL.observe` is always required. Thank you for your suggestion here!

> All of your algorithms use get_state to get information to make a decision on, right?

That's right. But I'm also planning to add more algorithms that rely on some other kinds of information.